ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -635,7 +635,16 @@ openstack-osuosl) ;;
 openstack-ppc64le) ;;
 openstack*) export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml ;;
 ovirt) export OVIRT_CONFIG="${SHARED_DIR}/ovirt-config.yaml" ;;
-nutanix) ;;
+nutanix)
+    if [[ -f "${CLUSTER_PROFILE_DIR}/prismcentral.pem" ]]; then
+        export SSL_CERT_FILE="${CLUSTER_PROFILE_DIR}/prismcentral.pem"
+    fi
+    if [[ -f "${CLUSTER_PROFILE_DIR}/proxy-conf.sh" ]]; then
+      # shellcheck disable=SC1091
+      source "${CLUSTER_PROFILE_DIR}/proxy-conf.sh"
```

Comment:
there is already an existing line for exporting proxy on https://github.com/openshift/release/blob/4f57fb766ee5e3aea3606b74fb1fae343aa98651/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L571, I think we should move this part of codes to `proxy-config-generate`